packages/esm-shr-app/src/shrpatient-summary/shrAllergiesDataTable.component.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from 'react';
+import styles from './shr-summary.scss';
+import { useTranslation } from 'react-i18next';
+import { DataTable, Table, TableHead, TableRow, TableHeader, TableBody, TableCell } from '@carbon/react';
+import { itemDetails } from '../types';
+import { usePagination } from '@openmrs/esm-framework';
+import { PatientChartPagination } from '@openmrs/esm-patient-common-lib';
+
+interface SHRDataTableProps {
+  data: itemDetails[];
```

Comment:
Preferred syntax is 
```suggestion
  data: Array<ItemDetails>
```
Its nit just to keep things consistent across repos

Reply:
Done